### PR TITLE
Add cljs tests

### PR DESCRIPTION
**Pull request by annapawlicka**
- Add test cljs build
- Update phantom.js test helpers
- Update cljs and om
- Update project.clj to use lein 2.5.0, new cljsbuild and new lein clean
  command
- Change structure of the project to look like this:
 |-- project.clj
 |-- src
 |   |-- clj
 |   |-- cljs
 |-- test
 |   |-- clj
 |   |-- cljs
 |-- resources
 | ...
- Update README.md with new build and test commands
- Update .travis.yml to run cljs tests

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 language: clojure
 lein: lein2
+script: lein2 cljsbuild test
```
Comment (otfrom) [inline .travis.yml]: Only runs cljs tests now and not all of them

File: phantom/phantomjs-shims.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+(function() {
```
Comment (otfrom) [inline phantom/phantomjs-shims.js]: Do we need these shims in the repo or can they be retrieved separately?

File: project.clj [hunk 3]
```diff
@@ -57,46 +72,36 @@
...
                  [org.clojure/tools.reader "0.8.8"]
 
-                 [liberator "0.12.2"]
-                 [cheshire "5.3.1"]
+                 ;; ClojureScript dependencies
 
-                 ;; Required for Cassandra (possibly only OSX)
-                 [org.xerial.snappy/snappy-java "1.1.1.3"]
+                 [org.clojure/clojurescript "0.0-2665" :scope "provided"]
+                 [org.clojure/core.async "0.1.346.0-17112a-alpha" :scope "provided"]
```
Comment (otfrom) [inline project.clj]: Is this the correct version pairing for cljs and core.async? I vaguely remember seeing some issues around this so I just want to check.

File: project.clj [hunk 4]
```diff
@@ -104,9 +109,7 @@
                               :port 3449
                               :css-dirs ["resources/site/css"]}
                    :env {:is-dev false}
-                   :plugins [[com.cemerick/austin "0.1.4"]
-                             [lein-figwheel "0.1.5-SNAPSHOT"]]}
-             :uberjar {:main kixi.hecuba.controller.main :aot [kixi.hecuba.controller.main]}}
+                   :plugins [[lein-figwheel "0.1.5-SNAPSHOT"]]}}
```
Comment (otfrom) [inline project.clj]: current version of lein-figwheel is 0.2.1-SNAPSHOT. Not sure if you want to upgrade this yet though

Comment (annapawlicka) [inline project.clj]: I'd rather do an upgrade of figwheel and other libraries as part of another PR.

File: project.clj (hunk 3 above)
Comment (annapawlicka) [inline project.clj]: I haven't seen any issues around this and I've done some testing locally.

File: phantom/phantomjs-shims.js (hunk 2 above)
Comment (annapawlicka) [inline phantom/phantomjs-shims.js]: They are not hosted anywhere and the usual approach is to copy the code to the repo.

File: .travis.yml (hunk 1 above)
Comment (annapawlicka) [inline .travis.yml]: I've fixed it by adding `lein test` to `script` command.

**Comment by otfrom** [comment]
:shipit:

**Pull request merged**